pkg/process/runner/runner.go

```diff
@@ -368,7 +368,12 @@ func (l *CheckRunner) basicRunner(c checks.Check) func() {
 			l.runCheck(c)
 		}
 
-		ticker := time.NewTicker(checks.GetInterval(l.config, c.Name()))
+		tickerInterval := checks.GetInterval(l.config, c.Name())
+		if c.Name() == checks.ConnectionsCheckName {
```

Comment:
Could you move this logic into `GetInterval()`? I see there's other check-specific logic in there already